spec/services/invoices/update_service_spec.rb

```diff
@@ -163,6 +166,19 @@
       end
     end
 
+    context 'with payment_status update and notification is turned on' do
+      let(:webhook_notification) { true }
+
+      it 'delivers a webhook' do
+        result
+
+        expect(SendWebhookJob).to have_been_enqueued.with(
+          'invoice.payment_status_updated',
+          invoice
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)